LinuxAddons/bin/restore_config

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+cp Marlin/Configuration.h.backup Marlin/Configuration.h
```

Comment:
`restore_config` could also restore `Configuration_adv.h` and  `pins_RAMPS_14.h`. Otherwise we will forget to restore these one day.